lib/tasks/technical_metadata.rake

```diff
@@ -33,4 +33,20 @@ namespace :techmd do
       end
     end
   end
+
+  namespace :reports do
+    desc 'Output a CSV of media durations for corresponding druids'
+    task media_durations: :environment do
+      results = File.foreach('druids.txt', chomp: true).flat_map do |druid|
+        DroFile.where(druid:)
+               .order(:filename)
+               .filter_map do |dro_file|
+          next if dro_file.av_metadata&.fetch('duration').blank?
+
```

Comment:
do not skip if blank? add in a filter by mimetype to avoid getting non media files?